### ISSUE TITLE
Cannot self-host playground in internal network

### ISSUE THREAD

**Issue by mesudip** [openfga/openfga#1174]
**Summary**
The playground is hosted at `http://172.31.6.0`, with nginx reverse proxy.

When I try to access the openfga playground, I get the following error

**Error message on browser console**
```
Refused to frame 'https://play.fga.dev/' because an ancestor violates the following 
Content Security Policy directive: "frame-ancestors http://localhost:*".
```

### Other details
- How are you running OpenFGA? (As a binary, in Docker, in Kubernetes)
On Docker
- What datastore are you using? (In memory, MySQL, Postgres)
Postgres
- Do you have any logs or traces that could help us debug the problem?
<img width="447" alt="image" src="https://github.com/openfga/openfga/assets/11131298/b7d8b1fc-cbc7-44ab-9dcc-37103163d723">

**Comment by Code2Life** [comment]
same issue

**Comment by ganarajtejasvi** [comment]
Works on localhost as it is allowed but blocked on self hosted endpoints

**Comment by rhamzeh** [comment]
Closing in favour of https://github.com/openfga/openfga/issues/338
